hibernate-reactive-core/src/test/java/org/hibernate/reactive/ORMReactivePersistenceTest.java

```diff
@@ -0,0 +1,187 @@
+/* Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ * Copyright: Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.reactive;
+
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistry;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.reactive.mutiny.Mutiny;
+import org.hibernate.reactive.provider.Settings;
+import org.hibernate.reactive.stage.Stage;
+import org.hibernate.reactive.testing.DatabaseSelectionRule;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import io.vertx.ext.unit.TestContext;
+
+import static org.hibernate.reactive.containers.DatabaseConfiguration.DBType.POSTGRESQL;
+import static org.hibernate.reactive.util.impl.CompletionStages.completedFuture;
+
+public class ORMReactivePersistenceTest extends BaseReactiveTest {
```

Comment:
I think in this case a brief javadoc explaining what this test is about is useful

Reply:
Something like this?

/**
 * These tests verify that data can be persisted and queried on the same database
 * using both JPA/hibernate and reactive Mutiny or CompletionStage session factories
 */

Reply:
Maybe something like this:
```
Test that it's possible to use an ORM and Reactive session in the same app
<p>
The test creates an entity using the ORM session and then retrieve it using the reactive one.
```

I don't have strong opinion, feel free to pick the one you prefer.